src/ports/postgres/modules/deep_learning/madlib_keras_model_selection.sql_in

```diff
@@ -288,7 +302,7 @@ load_model_selection_table(
 so we first create a model architecture table with two different models.  Use Keras to define
 a model architecture with 1 hidden layer:
 <pre class="example">
-import keras
+from tensorflow import keras
```

Comment:
we should use tensorflow.keras in the next two lines as well

Reply:
fixed